### ISSUE TITLE
[1.18.1] Custom Loot Tables are overwritten

### ISSUE THREAD

**Issue by Dradon7** [PlummetStudios/Insatiable-1.18#1]
I made a small datapack that made several animals drop a custom loot table for my modpack, and this mod overwrites this. This isn't a difficult thing to solve on my end, but for better mod compat (like if alex's mobs suddenly changed some loot tables) you should use loot injections instead of datapack overrides.

Basically, you can register various loot table additions like "villager_meat" that adds 0-3 strange meat to a loot table and then inject that into all the villager/illager mobs. The forge docs kinda suck, but I've linked them anyway cause you might find them helpful. [mcforge.readthedocs.io/en/1.18.x/items/globallootmodifiers/](mcforge.readthedocs.io/en/1.18.x/items/globallootmodifiers/)

Other mods like Farmers Delight use this to change the loot tables of vanilla animals.

This isn't that pressing of an issue, it just will prevent future problems and compat issues, and would probably make expanding to new mods slightly easier.

**Comment by PlummetStudios** [comment]
Yes! You are completely right, I completely forgot this feature! I will be making an update ASAP to implement this.

**Comment by PlummetStudios** [comment]
So from my knowledge, this would take a lot of new classes and time so maybe in a future update but probably not at this time as it would be over 60-70 classes that I would have to write by hand.
